mesa/visualization/templates/modular_template.html

```diff
@@ -139,12 +164,30 @@
 <script>
 	// Set up the GUI
 	var gui = new dat.GUI();
-	var fps_control = gui.add(control, "fps", 0, 20).step(1); // fps slider, 0 to 20 in 1 increments 
+	var fps_control = gui.add(control, "fps", 0, 50).step(1); // fps slider, 0 to 20 in 1 increments
```

Comment:
Ah, no not really. It just felt too slow for me sometimes ;)

Haven't tested it though, I wasn't aware of the buggy behavior. But that should at some time be fixed anyway, right? When you run a complex model on a slow computer it can already be buggy at 20 fps.

But I have nothing against leaving it at 20 for now.